scripts/run_lighthouse_tests_test.py

```diff
@@ -622,3 +645,38 @@ def mock_popen(*unused_args: str, **unused_kwargs: str) -> MockTask:  # pylint:
                                             '--mode', 'performance',
                                             '--skip_build',
                                             '--record_screen'])
+
+    def test_run_lighthouse_tests_dockerized(self) -> None:
+        class MockTask:
+            returncode = 0
+            def communicate(self) -> tuple[bytes, bytes]:   # pylint: disable=missing-docstring
```

Comment:
I suggest a brief docstring for this and other mocks to explain what scenario the mock is trying to simulate.

Reply:
I added one here; let me know if there should be others.